### PR TITLE
Enable different implementations libm

### PR DESCRIPTION
**Pull request by anton-bondarev**

### CODE REVIEW

**Comment by Deryugin** [comment]
If we use stubs for math operations, there's no need in `third_party.lib.NoOpenLibm` module (and `third_party.lib.OpenLibm` shold be non-abstract).

File: templates/msp430/exp430g2/mods.config [hunk 1]
```diff
@@ -17,7 +17,7 @@ configuration conf {
 	include embox.kernel.cpu.bkl_api
 
 	include embox.compat.libc.stdio.print(support_floating=0)
-	include third_party.lib.NoOpenLibm
+	include third_party.lib.libm_none
```
Comment (Deryugin) [inline templates/msp430/exp430g2/mods.config]: I think it should be `embox.compat.libc.math_libm_none` (same with other templates).

**Comment by anton-bondarev** [comment]
@Deryugin Thanks, I fixed it. Please look again.

File: src/compat/libc/math/stub.c [hunk 2]
```diff
@@ -0,0 +1,39 @@
+/**
+ * @file stub.c
```
Comment (AntonKozlov) [inline src/compat/libc/math/stub.c]: This implementation should not be used in any circumstances, right? Why should it exist?

Comment (anton-bondarev) [reply src/compat/libc/math/stub.c]: @AntonKozlov It's for future porting on E2K :)

**Pull request merged**